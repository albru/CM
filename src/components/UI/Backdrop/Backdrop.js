import React from 'react';
import classes from './Backdrop.css';
import Aux from '../../../hoc/_Aux/_Aux';

const backdrop = props => {
    const showBackdrop = props.isShow ? 
        <div 
            className={classes.Backdrop}
            onClick={props.close}>
        </div> : 
        null;
        
    return (
        <Aux>
            {showBackdrop}
        </Aux>
    )
};

export default backdrop;