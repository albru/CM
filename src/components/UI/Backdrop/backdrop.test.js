import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Backdrop from './Backdrop';

configure({ adapter: new Adapter() })

describe('Modal', () => {
    it('should render correctly', () => {
        const output = shallow(
            <Backdrop />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});