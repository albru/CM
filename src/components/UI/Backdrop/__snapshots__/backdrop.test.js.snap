// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop should render correctly 1`] = `<Aux />`;
